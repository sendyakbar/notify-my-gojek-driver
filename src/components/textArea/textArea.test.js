import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer'

import TextArea from './textArea.jsx'

describe('<TextArea> component', () => {
    const wrapper = shallow(<TextArea />)
    test('renders correctly', () => {
        expect(wrapper).toHaveLength(1)
    })
    test('it has a div', () => {
        const divWrapper = wrapper.find('div')
        expect(divWrapper).toHaveLength(1)
    })
    test('it has textarea', () => {
        const textareaWrapper = wrapper.find('textarea')
        expect(textareaWrapper).toHaveLength(1)
    })
    test('renders correctly and matched snapshot', () => {
        const tree = renderer.create(<TextArea />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})