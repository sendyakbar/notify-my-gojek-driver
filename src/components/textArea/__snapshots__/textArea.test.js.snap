// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextArea> component renders correctly and matched snapshot 1`] = `
<div>
  <textarea
    onChange={[Function]}
  />
</div>
`;
